### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by megan-godfrey**
Adds a nav bar
image

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
-    <%= yield %>
+    <%= render '/partials/navbar' %>
+    <div id="main">
+      <p class="notice"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
+      <%= yield %>
+    </div>
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
+    <script type="text/javascript" data-turbolinks-track='false' src="assets/application.js"></script>
```
Comment (Cena-Studio) [inline app/views/layouts/application.html.erb:24]: It looks like the `turbo-rails` has been commented out in `application.js`. Is turbo still affecting the pages in this case?

File: app/javascript/application.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
+// import "@hotwired/turbo-rails"
+// import "controllers"
```
Comment (Cena-Studio) [inline app/javascript/application.js:3]: May I know the reason to disable these two modules? I don't really understand why they were here though.

**Review by Cena-Studio — COMMENTED** [review]
The functionality is good. Some transitions will be added in the future.

Wanna figure out the reason for some implementations to improve my understanding before the branch is merged.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (megan-godfrey) [reply app/views/layouts/application.html.erb]: Line removed in commit below

**Review by Cena-Studio — APPROVED** [review]
LGTM. As discussed the dropdown will be further improved so the current possible bugs are known and would be fixed at a later moment.

**Pull request merged**